app/guide/notices.md

```diff
@@ -12,7 +12,7 @@ eleventyComputed:
   html: "Only <a href='/guide/users/#superusers'>superusers</a> can view important notices."
 }) }}
 
-Whenever you log in to Mavis, you should go to **Notices** to see if there’s anything you need to be aware of.
+Whenever you log in to Mavis, you should go to **Import** and then **Important Notices** to see if there’s anything you need to be aware of.
```

Comment:
Maybe we can mention that any important notices will be flagged up on the dashboard?